test/standalone/test-index.js

```diff
@@ -164,6 +164,36 @@ describe('index.js', function() {
     });
   });
 
+  describe('labels', function(){
+    it('should add labels to spans', function() {
+      agent.start();
+      cls.getNamespace().run(function() {
+        agent.private_().createRootSpanData('root', 1, 2);
+        var spanData = agent.startSpan('sub', {test1: 'value'});
+        agent.endSpan(spanData);
+        var traceSpan = spanData.span;
+        assert.equal(traceSpan.name, 'sub');
+        assert.ok(traceSpan.labels);
+        assert.equal(traceSpan.labels.test1, 'value');
+      });
+    });
+
+    it('should ignore non-object labels', function() {
+      agent.start();
+      cls.getNamespace().run(function() {
+        agent.private_().createRootSpanData('root', 1, 2);
+        var stringWithProperty = 'foo';
+        var spanData = agent.startSpan('sub', 'foo');
+        agent.endSpan(spanData);
+        var traceSpan = spanData.span;
+        assert.equal(traceSpan.name, 'sub');
+        assert.ok(traceSpan.labels);
+        assert.strictEqual(traceSpan.labels[0], undefined);
```

Comment:
Is there a reason this isn't a length check? I think the first label being `undefined` doesn't strictly capture the correctness condition.